### ISSUE TITLE
New-HPOVStorageVolumeTemplate with -LockProvisionMode

### ISSUE THREAD

**Issue by DungKHoang** [HewlettPackard/POSH-HPOneView#461]
the following code  does not 'lock' the 'shared'when creating a volume template  

```PowerShell
$stp = get-HPOVStoragePool -name "CPG-SSD-AO";new-HPOVStorageVolumeTemplate  -Name "svt-AO-3PAR"   -Capacity 3  -StoragePool $stp -Shared -LockProvisionMode
```

[2003091543_HPOV_transcript.log](https://github.com/HewlettPackard/POSH-HPOneView/files/4309304/2003091543_HPOV_transcript.log)

**Comment by ChrisLynchHPE** [comment]
The logic validation was incorrect within the Cmdlet.  I found and fixed the issue.  It will be fixed in the next release.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 5.00.2368.2592 ](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.00.2368.2592) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.500/5.00.2368.2592).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
